src/components/SmartBanner/index.tsx

```diff
@@ -238,39 +244,34 @@ class SmartBanner extends Component<Props> {
     this.show();
 
     const { icon, link, inStore } = this.retrieveInfo();
-    const wrapperClassName = `smartbanner smartbanner-${deviceType}`;
-    const iconStyle = {
-      backgroundImage: `url(${icon})`,
-    };
 
     return (
-      <div className={wrapperClassName}>
-        <div className="smartbanner-container">
-          <button
+      <Container className={`smartbanner ${deviceType}`}>
```

Comment:
is class `smartbanner` needed?

Reply:
it's needed to get controlled by the CSS class .smartbanner-show